tests/test_serializer_csv.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2016-2018 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""CSV Core serializer tests."""
+
+from __future__ import absolute_import, print_function
+
+from invenio_pidstore.models import PersistentIdentifier
+from invenio_records import Record
+from marshmallow import Schema, fields
+
+from invenio_records_rest.serializers.csv import CSVSerializer
+
+
+class SimpleSchema(Schema):
+    """Test schema."""
+
+    titles = fields.Raw(attribute='metadata.titles')
+    booklinks = fields.Raw(attribute='metadata.booklinks')
+    circulation = fields.Raw(attribute='metadata.circulation')
+    books = fields.Raw(attribute='metadata.books')
+    document_types = fields.Raw(attribute='metadata.document_types')
+    languages = fields.Raw(attribute='metadata.languages')
+    keywords = fields.Raw(attribute='metadata.keywords')
+    series_objs = fields.Raw(attribute='metadata.series_objs')
+
+    record_1 = {'titles': {"pid": 51, "other": 'A'},
```

Comment:
can you please add one or multiple values to test corner cases, for example:
* something like `'Jane " Doe'`
* something like `"Jane ' Doe"`
* something like `'Jane "aka" Doe'`
* something like `'François Doe'`
* any other you can think of